cmd/gf/internal/cmd/genpbentity/genpbentity.go

```diff
@@ -43,6 +44,7 @@ type (
 		Prefix            string `name:"prefix"            short:"f"  brief:"{CGenPbEntityBriefPrefix}"`
 		RemovePrefix      string `name:"removePrefix"      short:"r"  brief:"{CGenPbEntityBriefRemovePrefix}"`
 		RemoveFieldPrefix string `name:"removeFieldPrefix" short:"rf" brief:"{CGenPbEntityBriefRemoveFieldPrefix}"`
+		TableExclude      string `name:"tableExclude"      short:"te" brief:"{CGenPbEntityBriefTableExclude}"`
```

Comment:
Please use `TableEx` as the field name, to be the same as command `gen dao`: https://github.com/gogf/gf/blob/817ac36ce24af81b05e663a9d4f90e479b76d52a/cmd/gf/internal/cmd/gendao/gendao.go#L188

Reply:
done